csv/MFT.csv

```diff
@@ -1,5 +1,5 @@
 Detection,Filename,PathName,Reference
-Suspicious Location,\.(exe|dll|dit|msi|bat|rar|zip|dmp|ps1|7z)$,/Perflogs/|/ProgramData/|/Users/Public|/Windows/Temp|Appdata/Roaming/|Appdata/Local/|C:/Temp/,Internal
+Suspicious Location,.,(\\Perflogs\\|\\Users\\Public|\\Windows\\Temp|C:\\Temp\\).*\.(exe|dll|dit|msi|bat|rar|zip|dmp|ps1|7z|txt|vbs|vbe)$,,Internal
```

Comment:
@svch0stz from a performance viewpoint, its much better to filter by file first pass. I would reccomend moving the extentions back to the Filename field.